Tools/scripts/generate_opcode_h.py

```diff
@@ -75,9 +75,16 @@ def main(opcode_py, outfile='Include/opcode.h'):
             fobj.write(DEFINE.format(name, next_op))
             used[next_op] = True
         fobj.write(DEFINE.format('DO_TRACING', 255))
-        fobj.write("#ifdef NEED_OPCODE_JUMP_TABLES\n")
+        fobj.write("\nextern const uint8_t _PyOpcode_InlineCacheEntries[256];\n")
+        fobj.write("#ifdef NEED_OPCODE_TABLES\n")
```

Comment:
Minor, I just find this a bit easier to parse in the generated output:

```suggestion
        fobj.write("\n#ifdef NEED_OPCODE_TABLES\n")
```